ProjFS.Mac/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -553,7 +555,7 @@ void ActiveProvider_Disconnect(VirtualizationRootHandle rootIndex, PrjFSProvider
     RWLock_ReleaseShared(s_virtualizationRootsLock);
 }
 
-errno_t ActiveProvider_SendMessage(VirtualizationRootHandle rootIndex, const Message message)
+errno_t ActiveProvider_SendMessage(VirtualizationRootHandle rootIndex, const Message& message)
```

Comment:
Why did this change to a pass by reference?

Reply:
As `Message` becomes larger and larger, it'll be implicitly passed by reference anyway, but it'll be a reference to a copy. No need to copy, as we don't modify it anyway, so explicit readonly reference is in theory slightly more efficient.